packages/astro/package.json

```diff
@@ -28,22 +28,22 @@
   },
   "scripts": {
     "dev": "tsup --watch --onSuccess \"npm run build:dts\"",
-    "build": "tsup --onSuccess \"npm run build:dts\" && npm run copy:components",
+    "build": "tsup && npm run copy:components",
```

Comment:
Removing dts build here since we already use `dts` option of tsup. We can safely remove the exluded astro components folder in tsconfig

Reply:
Thanks for the link. I put it back and added a comment to the `ts` files we have in the astro components folder on why they're ignored. If we don't do it, `tsc` will throw errors.

We initially have the `astro-components` folder ignored in `tsconfig` but I removed it so that virtual modules can have types